### ISSUE TITLE
Store credentials in auth-source

### ISSUE THREAD

**Issue by parhamdoustdar** [org-trello/org-trello#416]
Hi there,

First of all, I wanted to give you a huge thank you. I'm completely blind and the Trello web interface is not accessible with my screen reader. However, thanks to your package I can use it pretty well! Thanks!

### What do you want?

Move authentication info to `auth-sources`.

### Why is it better?

Right now, the `consumer-key` and `access-token` are both stored as normal text. With a few changes, these could be moved to the `auth-sources` which areusually encrypted with no effort on the package maintainer's side.

**Comment by ardumont** [comment]
Thanks for the vote of confidence

And the pointer on the auth-source which is not somehing i knew.
